main.py

```diff
@@ -1,62 +1,65 @@
 from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
 from time import sleep
 from secrets import pw
 
 
 class InstaBot:
-    def __init__(self, username, pw):
-        self.driver = webdriver.Chrome()
-        self.username = username
-        self.driver.get("https://instagram.com")
-        sleep(2)
-        self.driver.find_element_by_xpath("//a[contains(text(), 'Log in')]")\
-            .click()
-        sleep(2)
-        self.driver.find_element_by_xpath("//input[@name=\"username\"]")\
-            .send_keys(username)
-        self.driver.find_element_by_xpath("//input[@name=\"password\"]")\
-            .send_keys(pw)
-        self.driver.find_element_by_xpath('//button[@type="submit"]')\
-            .click()
-        sleep(4)
-        self.driver.find_element_by_xpath("//button[contains(text(), 'Not Now')]")\
-            .click()
-        sleep(2)
+	def __init__(self, username, pw):
+		self.username = username
+		self.driver = webdriver.Chrome(ChromeDriverManager().install())
```

Comment:
will the rest of this code work on any version of code?

Reply:
The driver updates towards the beginning are universal, however this code was for a change insta made to the front-end for about a week. They constantly change it so I don’t expect it to work flawless as of today if that’s what you’re asking.